samples/graph-bulk-meetings/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-Meeting-Event-csharp",
+    "source": "officeDev",
+    "title": "This is a sample application that shows the usage of create training event meeting",
```

Comment:
This is a sample application which demonstrates how to create meetings in bulk for Teams.